test/AssetToken.js

```diff
@@ -132,6 +177,44 @@ contract('AssetToken', (accounts) => {
         assert.strictEqual(balanceRecepientFunded.toNumber() - defundVal, balanceRecepientDefunded.toNumber());
     });
 
+    it('transfer: Transfer more tokens than in the account', async () => {
+        const addrSender = accounts[1];
+        const addrRecepient = accounts[2];
+
+        const totalSupplyStart = await CONTRACT.totalSupply.call();
+        const balanceSenderStart = await CONTRACT.balanceOf.call(addrSender);
+        const balanceRecepientStart = await CONTRACT.balanceOf.call(addrRecepient);
+
+        const fundVal = 1;
+        CONTRACT.fund(addrSender, fundVal, { from: addrOwner });
```

Comment:
Is this an asynchronous or synchronous call? If asynchronous, why aren't we waiting for the result?

Reply:
Added an async